src/output-json.c

```diff
@@ -1242,10 +1241,16 @@ OutputInitResult OutputJsonInitCtx(ConfNode *conf)
     return result;
 
 error_exit:
-    if (json_ctx->file_ctx) {
-        LogFileFreeCtx(json_ctx->file_ctx);
-    }
     if (json_ctx) {
```

Comment:
why aren't we just removing this check? Its very clear that `json_ctx` can't be null here